src/App.tsx

```diff
@@ -5,7 +5,13 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie) => {
+    if (movies.every(element => element.imdbId !== movie.imdbId)) {
```

Comment:
I think, it would be great to show message for user why don't u add current movie to the list)